utils/test/RunningTests.md

```diff
@@ -0,0 +1,67 @@
+#Running tests for solutions-geoprocessing-toolbox
```

Comment:
If you instead called this file "Readme.md" - it would automatically be displayed when you navigate to `utils/test` folder (which would be "nice" behavior)